templates/db-template.yml

```diff
@@ -113,7 +113,7 @@ objects:
                     secretKeyRef:
                       key: database-name
                       name: ${DATABASE_SERVICE_NAME}
-              image: quay.io/stolostron/postgresql-13:1-101
+              image: quay.io/clyang82/postgresql-13:1-101
```

Comment:
why are we using any kind of private repo?  @gurnben why was this "stolostron" in the first place?

Seems like we should have some RH approved builds in an official RH org in Quay.

Reply:
That could be good if we can have an official RH org in Quay. But currently, we do not have yet. RH maintains the postgres image in registry.redhat.io. ACM needs the postgres in the upstream so ACM maintains a copy in quay.io/stolostron.

And we cannot rely on the docker.io due to the pull limit and I do not want to depend on `stolostron`. that is why I put them into my personal repos.  Do we have a plan to create the official quay.io/openshift-online repos so that we can maintain the images there?

Reply:
I have switched to use the official images. but you know we will meet `429 Too Many Requests - Server message: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit`. 

@markturansky @gurnben can we push the official images into quay.io/openshift-online for test purpose?